channels/us.m3u

```diff
@@ -6410,17 +6410,17 @@ https://lnc-witi-fox-aws.tubi.video/index.m3u8
 https://d2eit0bra9qkiw.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5765767dc/cmg_PROD_cmg-tv-10050_b9ac3c93-be86-4f7e-82b8-796b7f8bfda3_LE/in/cmg-wjaxtv-hls-v3/live.m3u8
 #EXTINF:-1 tvg-id="WJBK.us" tvg-name="FOX 2 Detroit (WJBK)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/EL9Bwtf.png" group-title="News",FOX 2 Detroit (WJBK) (720p)
 https://lnc-wjbk-fox-aws.tubi.video/index.m3u8
-#EXTINF:-1 tvg-id="WJLA.us" tvg-name="WJLA" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/LttuGfq.png" group-title="Local",WJLA
+#EXTINF:-1 tvg-id="WJLA.us" tvg-name="ABC 7 Washington D.C. (WJLA)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/LttuGfq.png" group-title="Local",ABC 7 Washington D.C. (WJLA)
 https://content.uplynk.com/40cec2bf074c40f08932da03ab4510be.m3u8
 #EXTINF:-1 tvg-id="WLNGRadio.us" tvg-name="WLNG Radio" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/11r6tgW.png" group-title="Local",WLNG Radio
 http://wlngstudiowebcam.srfms.com:1935/wlngstudiowebcam/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="WMFD.us" tvg-name="WMFD" tvg-country="US" tvg-language="English" tvg-logo="https://wmfd.com/images/logo.png" group-title="Local",WMFD (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="WMFDTV.us" tvg-name="WMFD-TV 68 Mansfield" tvg-country="US" tvg-language="English" tvg-logo="https://wmfd.com/images/logo.png" group-title="Local",WMFD-TV 68 Mansfield (720p) [Not 24/7]
 https://player-api.new.livestream.com/accounts/1194911/events/3466400/live.m3u8
-#EXTINF:-1 tvg-id="WMUR.us" tvg-name="WMUR New Hampshire" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",WMUR New Hampshire
+#EXTINF:-1 tvg-id="WMUR.us" tvg-name="ABC 9 New Hampshire (WMUR-TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",ABC 9 New Hampshire (WMUR-TV)
```

Comment:
I suppose you wanted to put WMURTV.us on tvg-id

```suggestion
#EXTINF:-1 tvg-id="WMURTV.us" tvg-name="ABC 9 New Hampshire (WMUR-TV)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6oNx5F0.png" group-title="Local",ABC 9 New Hampshire (WMUR-TV)
```

Reply:
Good catch